supported_formats_and_operations.md

```diff
@@ -1,12 +1,14 @@
 # Weight Formats
 
-The supported weight formats:
+The supported weight formats and their additional fields:
 
 - `keras_hdf5`: 
 - `pytorch_script`: A torchscript file.
 - `pytorch_state_dict`: A file containg the state dict of a pytorch model.
 - `tensorflow_js`: A zip file containing a json file and a binary weights file.
```

Comment:
Well, the files for tensorflow_js is not a zip file as we discussed.
```suggestion
- `tensorflow_js`: A text JSON file named model.json, which carries the topology and reference to the weights files, used by tensorflow.js.
```

Reply:
How is the weights file provided? Is it some URI that would need to be downloaded by the consumer?

Reply:
Tensorflow.js models are not meant for downloading but served directly from a server. So we only need to point to the URL of the json file, which will then automatically link to weights by the model loader.

Reply:
Ok, thanks for the clarification.